### PR TITLE
CardGroupListView: update Keys.forwardTo manually

### PR DESCRIPTION
**Pull request by nizovn**
usage of functions seems stopped to work for some reason.
Bug #1127.
Signed-off-by: Nikolay Nizov <nizovn@gmail.com>

### CODE REVIEW

File: qml/CardView/CardGroupListView.qml [hunk 1]
```diff
@@ -157,6 +157,7 @@ Item {
         }
 
         Keys.onPressed: {
+            updateKeysForwardTo();
```
Comment (Tofee) [inline qml/CardView/CardGroupListView.qml]: If I understand correctly, shouldn't we update the forwarding each time the current active window changes?

Comment (nizovn) [inline qml/CardView/CardGroupListView.qml]: right

Comment (Tofee) [inline qml/CardView/CardGroupListView.qml]: Maybe we could do that update here: https://github.com/webOS-ports/luna-next-cardshell/blob/master/qml/CardView/CardGroupListView.qml#L395

Comment (nizovn) [inline qml/CardView/CardGroupListView.qml]: Yes, thanks a lot. Seems I was too sleepy yesterday...

**Pull request merged**